### PR TITLE
adding constants and stripping units in restructuring

### LINKED ISSUE #103: outline restructuring
- why do this now
- how it is being done
- roadmap for next PRs

### LINKED ISSUE #104: set up environment utilities
- environment_ folder in utils
- contains mfp_air and dyn_vis_air
- associated tests

### LINKED ISSUE #105: add utility to strip units
- simple utility to return quantities with no units, akin to `.magnitude`.

### PR DESCRIPTION
**Pull request by ngam**
Fixes #105 
Fixes #104 
Fixes #103 

Follows #108 (should be reviewed and merged after 107 and 108, or it will close them)
Closes #107 
Closes #108 
Closes #99 

Highlights:
- a simple utility to strip units
- uses `.magnitude` method from pint but also avoids throwing en error if the input has no units
- useful if desired to strip units

### CODE REVIEW

**Comment by Gorkowski** [comment]
this is good to have :)

**Review by Gorkowski — APPROVED** [review]

**Review by wkchuang — APPROVED** [review]

**Comment by ngam** [comment]
@aktwo i implemented your suggestions from the other PR. Could you please review and merge? This will close all associated issues and PRs. And we go back to one-review-per-PR from now onward. Thanks!

**Comment by aktwo** [comment]
Ported over some related comments from other PRs! Lmk what you think

**Comment by ngam** [comment]
> Ported over some related comments from other PRs! Lmk what you think

I am not following...

**Comment by ngam** [comment]
Alright, I am going to move on; @aktwo, if you have any issues and other suggestions, please feel free to submit PRs

**Comment by aktwo** [comment]
Oh whoops sorry I'm not used to this UI on my phone, I think I must have forgotten to click send! No worries, they weren't a huge deal :)

**Comment by ngam** [comment]
No worries, I will incorporate them in upcoming PRs. The thing with `get_constants` vs `constants` is that I thought the former is more descriptive. I can go either way. With `strip_units`, we could do `units.py` and then in it `strip` function. Let me figure that out in time for more PRs this evening.

File: particula/tests/unitless_test.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+""" A quick test of strip_units.unitless utility
```
[the remark this answers is not in the document]
Comment (ngam) [reply particula/tests/unitless_test.py:1]: ✅

File: particula/strip_units.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+""" A simple utility to get rid of units if desired.
```
[the remark this answers is not in the document]
Comment (ngam) [reply particula/strip_units.py:1]: ✅

File: particula/get_constants.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+""" A centralized location for important, unchanged physics parameters.
+
+    This file contains constants that are used in multiple modules. Each
+    constant has its own units and exported with them. The constants are
+    mostly related to atmospheric aerosol particles in usual conditions.
+    They are flexible enough to depend on on the temperature and other
```
[the remark this answers is not in the document]
Comment (ngam) [reply particula/get_constants.py:6]: ✅

File: particula/aerosol_dynamics/physical_parameters.py [hunk 4]
```diff
@@ -1,29 +1,14 @@
...
-# permittivity of free space in F/m
-# Also known as the electric constant, permittivity of free space
-# Often denoted by epsilon_0
-VACUUM_PERMITTIVITY = 8.85418782e-12 * u.F / u.m
-
-# permittivity of air
-ELECTRIC_PERMITTIVITY = RELATIVE_PERMITTIVITY_AIR * VACUUM_PERMITTIVITY
+# pylint: disable=unused-import,
+# flake8: noqa: F401
+from particula.get_constants import (
```
[the remark this answers is not in the document]
Comment (ngam) [reply particula/aerosol_dynamics/physical_parameters.py:6]: ✅

File: particula/get_constants.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    constants:
+
+    TODO:
+        - Add a way to override the constants by parsing a user-supplied
+            configuration file.
+"""
+
+from particula import u
+
+# Boltzmann's constant in m^2 kg s^-2 K^-1
```
[the remark this answers is not in the document]
Comment (ngam) [reply particula/get_constants.py:18]: ✅ 

(left some more noteworthy comments that are not directly decipherable from the code itself)

**Comment by ngam** [comment]
> Oh whoops sorry I'm not used to this UI on my phone, I think I must have forgotten to click send

This happens to me all the time --- it gives you the illusions that your comments have been submitted, but you actually have to submit the whole review independently (after-the-fact) for them to show up. So misleading!

**Pull request merged**